.gitignore

```diff
@@ -79,3 +59,12 @@ eclipse-projects/*/.tmpBin/
 # FOR VIM
 #
 .*.swp
+#
+# For Eclipse
+#
+.classpath
+.project
+bin/    
+/eclipse-projects
+.settings/
```

Comment:
Are ``bin/`` and ``/eclipse-projects`` needed anymore? I ran the sbt command in the commit message and neither of these directories exist. Or are these created when you import/compile with eclipse?

Reply:
/bin by default is where eclipse puts compiled objects. I put /eclipse-projects in just in case somebody wants to maintain the old project structure or switch back and forth with prior releases that have this directory in it.

Reply:
Reminder: Add doc to .gitignore indicating that we don't want IDE artifacts in the source tree anymore if avoidable. Creating them from the sbt information is preferred, as is done by IDEA and the sbt eclipse plugin.